packages/documentation/src/stories/foundation/icons/icons.stories.mdx

```diff
@@ -0,0 +1,288 @@
+
+<Canvas withSource="open">
+  <Story
+    name="Scaling"
+    decorators={[
+      Story => (
+        <div className="d-flex flex-wrap gap-3" style={{ fontSize: '32px' }}>
+          <Story />
+        </div>
+      ),
+    ]}
+  >
+    <post-icon name="3200" class="bg-info" scale="0.5" />
+    <post-icon name="3200" class="bg-info" />
+    <post-icon name="3200" class="bg-info" scale="1.5" />
+  </Story>
+</Canvas>
+
+#### Rotating
+
+Rotate the icon by a number of degrees with the `rotate` prop. Positive values will rotate the icon clockwise, while negative values will rotate the icon counterclockwise.
+
+<Canvas withSource="open">
+  <Story
+    name="Rotating"
+    decorators={[
+      Story => (
+        <div className="d-flex flex-wrap gap-2" style={{ fontSize: '32px' }}>
+          <Story />
+        </div>
+      ),
+    ]}
+  >
+    <post-icon name="1001" rotate="-360" />
+    <post-icon name="1001" rotate="-249" />
+    <post-icon name="1001" rotate="-35.5" />
+    <post-icon name="1001" rotate="0" />
+    <post-icon name="1001" rotate="98" />
+    <post-icon name="1001" rotate="360" />
+    <post-icon name="1001" rotate="753" />
+  </Story>
+</Canvas>
+
+#### Animating
+
+The `post-icon` includes the following built-in animations for icons:
+
+- `cylon` slides the icon left and right
+- `cylon-vertical` slides the icon up and down
+- `spin` smoothly spins the icon clockwise
+- `spin-reverse` smoothly spins the icon counter-clockwise
+- `fade` fades the icon in and out
+- `throb` scales the icon in and out
+
+To use them, set the `animation` prop to one of the animation names above.
+
+<Canvas withSource="open">
+  <Story
+    name="Animating"
+    decorators={[
+      Story => (
+        <div className="d-flex flex-wrap text-center" style={{ fontSize: '32px' }}>
+          <Story />
+        </div>
+      ),
+    ]}
+  >
+    <div className="w-50 py-3">
+      <p className="text-muted fs-tiny">Cylon animation:</p>
+      <post-icon name="2253" animation="cylon" />
+    </div>
+    <div className="w-50 py-3">
+      <p className="text-muted fs-tiny">Vertical cylon animation:</p>
+      <post-icon name="2252" animation="cylon-vertical" />
+    </div>
+    <div className="w-50 py-3">
+      <p className="text-muted fs-tiny">Spinning animation:</p>
+      <post-icon name="2041" animation="spin" />
+    </div>
+    <div className="w-50 py-3">
+      <p className="text-muted fs-tiny">Reverse spinning animation:</p>
+      <post-icon name="2042" animation="spin-reverse" />
+    </div>
+    <div className="w-50 py-3">
+      <p className="text-muted fs-tiny">Fade animation:</p>
+      <post-icon name="1028" animation="fade" />
+    </div>
+    <div className="w-50 py-3">
+      <p className="text-muted fs-tiny">Throb animation:</p>
+      <post-icon name="2063" animation="throb" />
+    </div>
+  </Story>
+</Canvas>
```

Comment:
We should move this to the post-icon component documentation

Reply:
Moved this to a separate issue #1083